11-multi-track/bb_models_collections/readme.md

```diff
@@ -318,9 +361,9 @@ Collections can use the following:
 Look at the repo [here](https://github.com/ga-dc/reminder#part-1)
 
 ## AJAX /w BB (60/90)
-So we've learned how to create backbone models and collections, but it'd be really nice if we could define our models/collections through an existing backend. Enter rails. But it doesn't have to be! But for the purposes of this class we'll use a nice and simple rails application that we've created for you.
+So we've learned how to create Backbone Models and Collections, but so far we have been hard-coding our data, instead it would be great if we could fetch the data for our models/collections through an existing backend. For the purposes of this class, we'll use a nice and simple Rails application that we've created for you.
 
-You can pull the code for our backend [here](https://github.com/ga-dc/grumblr_rails_api)
+You can pull the code for our backend API [here](https://github.com/ga-dc/grumblr_rails_api)
 
```

Comment:
I remember during the last cohort people were trying to add a Backbone gem to the Rails app and getting confused by it. Not sure why, but wanted to make a note of it in case you feel it's necessary to mention in class.